src/config.inc

```diff
@@ -10,10 +10,10 @@ endif
 
 # Select optimisation or debug info
 #CXXFLAGS += -O2 -DNDEBUG
-#CXXFLAGS += -O0 -g
+CXXFLAGS += -O0 -g
 
 # With GCC this adds function names in stack backtraces
-#LINKFLAGS="-rdynamic"
+#LINKFLAGS=-rdynamic
```

Comment:
If we're doing this, can we also add spaces around the `=`?

Reply:
+1

Reply:
Sure.